### PR TITLE
Outer Port Vision Targeting

### LINKED ISSUE #28: Subsystem: Vision System
Production software for the vision subsystem in the robot code, including:
- interface definitions (i.e. network tables)
- commands
- etc..
- tested independently
- merged and integrated with 2020 robot code in github
- ready for integration

### PR DESCRIPTION
**Pull request by Zinka010**
- Command to allow turning to the outer port based on yaw provided by the vision
- Various improvements and bulletproofing to DriveTrainPIDTurnDelta
- Some refactoring of general  variable names

### CODE REVIEW

**Comment by Zinka010** [comment]
Note that the majority of changes come from my compulsive need to autoformat everything

**Review by georgetzavelas — COMMENTED** [review]
Looks like you need a rebase. 
I would highly advise you to consider shipping fewer changes in your PRs as too many changes are hard to give a good review to and also increase the likelihood of introducing a bug.

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -85,6 +82,9 @@ private void configureButtonBindings() {
 
         sensitiveDriverControlCommand = new SensitiveDriverControl(driverStick);
 
+        JoystickButton turnToYaw = new JoystickButton(driverStick, XboxValue.XBOX_A_BUTTON.getPort());
+        turnToYaw.whenPressed(new TurnToOuterPortCommand(true, 3.0, 1.0));
```
Comment (georgetzavelas) [inline src/main/java/frc/robot/RobotContainer.java]: What are 3.0 and 1.0? Can we make these constants? If we feel this is the only time we will instantiate this class then maybe we can make these defaults in the construtor?

**Review by AbhiJ2706 — APPROVED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (Zinka010) [reply src/main/java/frc/robot/RobotContainer.java]: So these are values which need to be tuned later so I'll make them into FluidConstants

**Review by georgetzavelas — APPROVED** [review]

**Comment by Zinka010** [comment]
Closes #102

**Pull request merged**